src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -480,6 +491,45 @@ public UserStatusInfo getUserInfo(AuthenticatedUserRequest userReq) {
     }
   }
 
+  @Override
+  public RepositoryStatusModelSystems samStatus() {
+    try {
+      return SamRetry.retry(
+          configurationService,
+          () -> {
+            StatusApi samApi = samStatusApi();
+            SystemStatus status = samApi.getSystemStatus();
+            return new RepositoryStatusModelSystems()
+                .ok(status.getOk())
+                .message(status.getSystems().toString());
+          });
+    } catch (Exception ex) {
+      String errorMsg = "Sam status check failed";
+      logger.error(errorMsg, ex);
+      return new RepositoryStatusModelSystems().ok(false).message(errorMsg + ": " + ex.toString());
```

Comment:
unnecessary `.toString()`
```suggestion
      return new RepositoryStatusModelSystems().ok(false).message(errorMsg + ": " + ex));
```

Reply:
Oh, right, I didn't even look in this method.  I was just moving it so it was lining up with the interface order.  I'll update